operations/boot_orchestration/Tools_for_Resolving_Boot_Issues.md

```diff
@@ -2,6 +2,27 @@
 
 A number of tools can be used to analyze and debug issues encountered during the compute node boot process. The underlying issue and symptoms dictate the type of tool required.
 
+## BOS
+
+BOS component records offer a history of operations that have been applied to a node.
+
+```bash
+cray bos components describe x3000c0s19b4n0
+```
+
+Additionally, nodes that are unable to boot may also be disabled. BOS will not apply power actions to disabled nodes,
+allowing an administrator to debug the node environment interactively.
```

Comment:
Do we need to say anything about the node being disable, so long as it does not become a part of a new session?

I suppose an alternate way to disable nodes is to do so in HSM. That would also keep the node from being operated on. I think it's more dangerous because HSM disabling has to be manually undone whereas just making the node a part of a new BOS session re-enables it. From the debugger's point of view though, the reverse is true. HSM-disablement is safer because it cannot be accidentally undone.

Reply:
I had attempted to view the problem as what we could accomplish in terms of cfs specifically, but this is probably the fine grained hammer that we need. Besides; not to hash up previous regretful skeletons, but HSM's disable flag in the PCS world is on the honor system 😆

Reply:
Doesn't BOS always honor it though, or did we change that when we converted to PCS?

I missed the connection between this documentation and it being specifically related to CFS. Could you explain?